collector/lib/Logging.h

```diff
@@ -55,7 +55,10 @@ bool ParseLogLevelName(std::string name, LogLevel* level);
 const char* GetGlobalLogPrefix();
 void SetGlobalLogPrefix(const char* prefix);
 
+void WriteTerminationLog(std::string message);
+
 const size_t LevelPaddingWidth = 7;
+const std::string TerminationLog = "/dev/termination-log";
```

Comment:
Do we need a full-on string? Can we use a raw `char*` instead?

Reply:
I had to move this into `Logging.cpp` to get the linker to stop complaining about redefinitions once I changed the type but it's probably a better place for it anyway